lib/pip.js

```diff
@@ -610,10 +610,9 @@ function installAllRequirements() {
       !fse.existsSync(symlinkPath) &&
       reqsInstalledAt != symlinkPath
     ) {
-      // Windows can't symlink so we have to copy on Windows,
-      // it's not as fast, but at least it works
+      // Windows can't symlink so we have to use junction on Windows
       if (process.platform == 'win32') {
-        fse.copySync(reqsInstalledAt, symlinkPath);
+        fse.symlink(reqsInstalledAt, symlinkPath, 'junction');
```

Comment:
Does this require any extra user permissions? I recall something about that

Reply:
It is my understanding that junctions only require the normal file creation permissions. When created they use fully qualified paths and can only link to directories in the same NTFS filesystem.

The admin permissions are required for a full symlink because this has more serious security complications.

The only potential problem I can see is if someone has set up their cache directory and the serverless directory on separate file systems. In that case the junction operation would fail. I could add this note to the readme, but it may be a lot of noise for an unlikely setup.

Reply:
Oh, very cool. I thought junctions _were_ symlinks. Thanks for the clarification.